docs/en/stack/ml/anomaly-detection/ootb-ml-jobs-siem.asciidoc

```diff
@@ -382,6 +382,197 @@ they are listed for each job.
 |https://github.com/elastic/kibana/blob/{branch}/x-pack/plugins/ml/server/models/data_recognizer/modules/security_windows/ml/v3_windows_rare_user_type10_remote_login.json[image:images/link.svg[A link icon]]
 |https://github.com/elastic/kibana/blob/{branch}/x-pack/plugins/ml/server/models/data_recognizer/modules/security_windows/ml/datafeed_v3_windows_rare_user_type10_remote_login.json[image:images/link.svg[A link icon]]
 
+|===
+// end::security-windows-jobs[]
+
+[discrete]
+[[security-integrations-jobs]]
+== Security: Elastic Integrations
+
+https://docs.elastic.co/integrations[Elastic Integrations] are a streamlined way to add Elastic assets to your environment, such as data ingestion, transforms, and in this case, machine learning capabilities for Security.
+
+The following Integrations use {ml} to analyze patterns of user and entity behavior, and help detect and alert when there is related suspicious activity in your environment.
+
+* https://docs.elastic.co/integrations/ded[Data Exfiltration Detection]
+* https://docs.elastic.co/integrations/dga[Domain Generation Algorithm Detection]
+* https://docs.elastic.co/integrations/lmd[Lateral Movement Detection]
+* https://docs.elastic.co/integrations/problemchild[Living off the Land Attack Detection]
+
+// dga
+
+*Domain Generation Algorithm (DGA) Detection*
+
+ML solution package to detect domain generation algorithm (DGA) activity in your network data. Requires a Platinum subscription.
+
+To download, see the https://docs.elastic.co/integrations/dga[documentation].
```

Comment:
The preferred term to use is `refer` instead of `see` for [accessibility reasons](https://docs.elastic.dev/content-design/accessibility#terms).
```suggestion
To download, refer to the https://docs.elastic.co/integrations/dga[documentation].
```